CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- Match `.luau` filename extension by default.
+- Allow multiple comma-separated glob patterns with `--patterns` rather than `--pattern`.

```

Comment:
`--pattern` should just take multiple patterns, I don't want to break backwards compatibility with the older parameter, and commas already aren't allowed in filenames in any of the operating systems we support, from what I know

Reply:
How would I do that?

Reply:
Just don't change the name :P

Reply:
Oh, I thought you were suggesting something like `selene --pattern foo --pattern foo`.